### PR TITLE
Select Attribute UI

### PR DESCRIPTION
**Pull request by techeng322**

### CODE REVIEW

File: components/Pages/CreatePage/AttributeButtons/AttributeButtons.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <div
+    className="flex flex-col 
+    md:gap-y-[6px] lg:gap-y-[8px] xl:gap-y-[10px]
+    md:px-[18px] lg:px-[24px] xl:px-[30px]"
+  >
+    <Button value="neckaceGolo" label="Acc." />
+    <Button value="red" label="Eyes" />
+    <Button value="horn" label="Hair" />
+    <Button value="Brown" label="Color." />
+    <Button value="YellowGeneric" label="Dutfit" />
```
Comment (sweetmantech) [inline components/Pages/CreatePage/AttributeButtons/AttributeButtons.tsx]:
```suggestion
    <Button value="YellowGeneric" label="Outfit" />
```

File: components/Pages/CreatePage/AttributeButtons/AttributeButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const AttributeButtons = () => (
+  <div
+    className="flex flex-col 
+    md:gap-y-[6px] lg:gap-y-[8px] xl:gap-y-[10px]
+    md:px-[18px] lg:px-[24px] xl:px-[30px]"
+  >
+    <Button value="neckaceGolo" label="Acc." />
+    <Button value="red" label="Eyes" />
+    <Button value="horn" label="Hair" />
+    <Button value="Brown" label="Color." />
```
Comment (sweetmantech) [inline components/Pages/CreatePage/AttributeButtons/AttributeButtons.tsx]:
```suggestion
    <Button value="Brown" label="Color" />
```

File: components/Pages/CreatePage/AttributeButtons/AttributeButtons.tsx (hunk 1 above)
Comment (techeng322) [reply components/Pages/CreatePage/AttributeButtons/AttributeButtons.tsx]: fixed ✅

File: components/Pages/CreatePage/AttributeButtons/AttributeButtons.tsx (hunk 2 above)
Comment (techeng322) [reply components/Pages/CreatePage/AttributeButtons/AttributeButtons.tsx]: fixed ✅

**Review by sweetmantech — APPROVED** [review]
ship it 🚢

**Pull request merged**